### PR TITLE
Vale exploratory work

### PR DESCRIPTION
**Pull request by jason-seqera**
Explores using Vale on either built HTML or on MDXv3 files with creatively crafted skip and ignore rules for unrecognized MDX features.

### CODE REVIEW

File: .github/styles/config/vocabularies/Seqera/accept.txt [hunk 1]
```diff
@@ -41,8 +41,8 @@ FSx
 Dockerfile
 initContainers
 Cron
-Autoscaling
-Autoscale
+[Aa]utoscaling
+[Aa]utoscale
```
Comment (christopher-hakkaart) [inline .github/styles/config/vocabularies/Seqera/accept.txt:45]: Do we need to add things like Fusion, MultiQC, and Wave here? Especially if we are treating them as proper nouns?

Comment (jason-seqera) [reply .github/styles/config/vocabularies/Seqera/accept.txt:45]: @christopher-hakkaart, probably eventually; it only triggers if a file in a given PR has a violation, so we don't need to accumulate all the possibilities before merging.

Comment (christopher-hakkaart) [reply .github/styles/config/vocabularies/Seqera/accept.txt:45]: All good 👍

**Review by christopher-hakkaart — APPROVED** [review]
No blockers from me - looks great

**Pull request merged**